release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -145,6 +145,15 @@ With this release, you can configure an {product-title} cluster on any on-premis
 
 For more information about this feature on bare-metal infrastructure, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#nw-osp-services-external-load-balancer_ipi-install-installation-workflow[Services for a user-managed load balancer] in _Setting up the environment for an OpenShift installation_.
 
+[id="ocp-4-16-networking-pathing-existing-dual-stack"]
+==== Patching an existing dual-stack network
+
+With this release, you can add IPv6 virtual IPs (VIPs) for API and Ingress services to an existing dual-stack-configured cluster by patching the cluster's infrastructure.
+
+If you have already upgraded your cluster to {product-title} 4.16 and you need to convert the single-stack cluster network to a dual-stack cluster network, you must specify an IPv4 network for API and Ingress services for your cluster in the YAML configuration patch file.
```

Comment:
We assume the cluster was installed as IPv4 and for dual-stack we are adding an IPv6 network for API and Ingress, so this should be "you must specify an IPv6 network"

Reply:
I don't think that's what was meant here. The problem with pre-4.16 born clusters is that you install them as dual-stack IPv4 and now you want to add 2nd pair of VIPs. So during the patching process you will need to specify IPv6 VIP for API, IPv6 VIP for Ingress, IPv6 network for 2nd machine network **and now important part**, also IPv4 network for 1st machine network.

Just because we never have access to the original install-config yaml to get the first machine network, users from pre-4.16 need to specify it manually.

If your cluster was born as 4.16 and you only want to add 2nd pair of VIPs, then you don't need to touch IPv4 at all because during installation we copied the IPv4 machine network aside.

Reply:
Hi @rbbratta . I updated the note to state:

* An IPv4 network for API and Ingress services on the first `machineNetwork`.
* An IPv6 network for API and Ingress services on the second `machineNetwork`.

WDYT?